web-ui/src/pages/CheckinsPage.jsx

```diff
@@ -13,8 +13,10 @@ const CheckinsPage = () => {
   const [show, setShow] = useState(false);
   const { state } = useContext(AppContext);
   const { checkins, userProfile } = state;
+  console.log({ userProfile });
   const [index, setIndex] = useState(0);
   const checkin = checkins[index];
+  const canSeePersonnel = userProfile && userProfile.role.includes("PDL");
```

Comment:
can ADMIN also view it ? do we need another check for ADMIN role ?

Reply:
The story said only if the user is a PDL, but maybe we do want admin to see it too? @mkimberlin What do you think?

Reply:
Only PDLs. If an ADMIN is also a PDL, yes... If not, then it doesn't make sense for them to see it.